### PR TITLE
Updating SDK docs to include new genesis generation

### PR DESCRIPTION
**Pull request by IPadawans**
### Changes
+ Fixing some typos
+ Updating genesis process to match the new genesis functions: `create-genesis` and `run-genesis`
+ Adding instruction to run data-l1

### CODE REVIEW

File: sdk/guides/setup-a-metagraph/building-metagraph-instances/02-building-currency-l1-instances.md [hunk 1]
```diff
@@ -0,0 +1,192 @@
+---
+id: building-currency-l1-instances
+title: Building currency L1 Instances
+---
+
+
+Let’s create our Currency L1 😎, our L1 will need you to set up at least 3 instances.
```
Comment (AlexBrandes) [inline sdk/guides/setup-a-metagraph/building-metagraph-instances/02-building-currency-l1-instances.md:7]: It seems like this is describing creating 3 independent servers to host the currency L1 nodes. Don't the rest of the instructions assume that all of the layers are hosted on the same 3 servers?

File: sdk/guides/setup-a-metagraph/building-metagraph-instances/03-building-data-l1-instances.md [hunk 2]
```diff
@@ -1,15 +1,19 @@
 ---
-id: building-metagraph-L1-instances
-title: Building metagraph L1 Instances
+id: building-data-l1-instances
+title: Building data L1 Instances
 ---
 
 
-Let’s create our Metagraph L1 😎, our L1 will need you to set up at least 3 instances. 
+Let’s create our Data L1 😎, our L1 will need you to set up at least 3 instances.
```
Comment (AlexBrandes) [inline sdk/guides/setup-a-metagraph/building-metagraph-instances/03-building-data-l1-instances.md:7]: Same here. Are we creating all the layers on the same 3 servers or are we assuming the projects will create 3x4 layers = 12 individual servers?

File: sdk/guides/setup-a-metagraph/01-overview.md [hunk 3]
```diff
@@ -24,12 +24,12 @@ Please note that all steps in this tutorial are performed on the EC2 page within
 
 **2. [Creating key pairs](/sdk/guides/setup-a-metagraph/key-pairs)**
 
-**3. [Generating base instance](/sdk/guides/setup-a-metagraph/generating-base-instance)**
+**3. [Generating base instance](/sdk/guides/setup-a-metagraph/base-instance/generating-base-instance)**
```
[the remark this answers is not in the document]
Comment (IPadawans) [reply sdk/guides/setup-a-metagraph/01-overview.md:27]: Will do

File: sdk/guides/setup-a-metagraph/base-instance/02-configuring-base-instance.md [hunk 4]
```diff
@@ -0,0 +1,185 @@
...
+## Base instance setup
+
+- Now, you can start setting up your instance. Create a directory named **`code`** and navigate into it:
+
+```bash
+mkdir code
+cd code/
+```
+
+- Additionally, let's create the following directories: `metagraph-l0`, `currency-l1`, and `data-l1`
```
[the remark this answers is not in the document]
Comment (IPadawans) [reply sdk/guides/setup-a-metagraph/base-instance/02-configuring-base-instance.md:46]: Fixed

File: sdk/guides/setup-a-metagraph/base-instance/02-configuring-base-instance.md [hunk 5]
```diff
@@ -0,0 +1,185 @@
...
+- For the metagraph execution you need to generate the metagraph JARs. There are two ways to obtain these JARs: 
+
+  - [Generating JARs on Base Instance](/sdk/guides/setup-a-metagraph/base-instance/generating-metagraph-jars)
+	-	[Generating JARs with the Euclid SDK](/sdk/guides/setup-a-metagraph/base-instance/using-jars-from-euclid)
+  
+
+## Setting up the genesis file
+
+:::info
+The Genesis file is a crucial component of your Metagraph. Thats where your metagraphID is generated and where you generate the initial token balances for your initial distributions.
```
[the remark this answers is not in the document]
Comment (IPadawans) [reply sdk/guides/setup-a-metagraph/base-instance/02-configuring-base-instance.md:94]: I think Joao as edit this part, I've removed the metagraphId part

File: sdk/guides/setup-a-metagraph/base-instance/02-configuring-base-instance.md [hunk 6]
```diff
@@ -0,0 +1,185 @@
...
+:::
+
+### Dependencies
+- Start by downloading some files from the Tessellation repo releases. Go to the releases [https://github.com/Constellation-Labs/tessellation/releases/tag/v2.2.0](https://github.com/Constellation-Labs/tessellation/releases/tag/v2.2.0)
```
[the remark this answers is not in the document]
Comment (IPadawans) [reply sdk/guides/setup-a-metagraph/base-instance/02-configuring-base-instance.md:98]: fixed

File: sdk/guides/setup-a-metagraph/base-instance/02-configuring-base-instance.md [hunk 7]
```diff
@@ -0,0 +1,185 @@
...
+mv cl-keytool.jar data-l1/cl-keytool.jar
+
+mv cl-node.jar global-l0/global-l0.jar
+```
+
+:::warning 
+Make sure you're using the latest version of Tessellation. You can find the most recent release in [**here**](https://github.com/Constellation-Labs/tessellation/releases).
+:::
+
+### Creating Genesis file
```
[the remark this answers is not in the document]
Comment (IPadawans) [reply sdk/guides/setup-a-metagraph/base-instance/02-configuring-base-instance.md:122]: Updated

File: sdk/guides/setup-a-metagraph/base-instance/02-configuring-base-instance.md [hunk 8]
```diff
@@ -0,0 +1,185 @@
...
+
+:::info
+The steps below use vim, you can use any text editor of your preference
+:::
+
+```bash
+vim genesis.csv
+```
+
+- Add the following values to the file:
```
[the remark this answers is not in the document]
Comment (IPadawans) [reply sdk/guides/setup-a-metagraph/base-instance/02-configuring-base-instance.md:140]: Updated

File: sdk/guides/setup-a-metagraph/base-instance/02-configuring-base-instance.md [hunk 9]
```diff
@@ -0,0 +1,185 @@
...
+DAG4Zd2W2JxL1f1gsHQCoaKrRonPSSHLgcqD7osU,1000000000000
+```
+
+:::info
+Edit the addresses and balances according to your needs or leave the file empty if your Metagraph starts without any token distribution.
+:::
+
+### Generating metagraphID
+
+-Now that you have generated your genesis file, the next step is to generate the metagraphID. To accomplish this, execute the following command:
```
[the remark this answers is not in the document]
Comment (IPadawans) [reply sdk/guides/setup-a-metagraph/base-instance/02-configuring-base-instance.md:154]: Updated

File: sdk/guides/setup-a-metagraph/base-instance/02-configuring-base-instance.md [hunk 10]
```diff
@@ -0,0 +1,185 @@
...
+- The environment variables above are required, but they will be overwritten in the next steps
+ 
+- This command should complete successfully, and you should find the following files in your directory:
+  - `genesis.snapshot`
+  - `genesis.address`
+
+- You can open the `genesis.address` file to view your metagraphID, which should resemble a DAG address: `DAG...`
+
+:::important
+The MetagraphID needs to be shared with the Constellation Network team to be added to the Allowance List
```
[the remark this answers is not in the document]
Comment (IPadawans) [reply sdk/guides/setup-a-metagraph/base-instance/02-configuring-base-instance.md:182]: Updated

File: sdk/guides/setup-a-metagraph/base-instance/02-configuring-base-instance.md [hunk 11]
```diff
@@ -0,0 +1,185 @@
...
+:::
+
+The base instance is now completely configured and you will use it in all Metagraph instances
```
[the remark this answers is not in the document]
Comment (IPadawans) [reply sdk/guides/setup-a-metagraph/base-instance/02-configuring-base-instance.md:185]: updated

File: sdk/guides/setup-a-metagraph/base-instance/03-generating-metagraph-jars.md [hunk 12]
```diff
@@ -0,0 +1,103 @@
+---
+id: generating-metagraph-jars
+title: Generating metagraph JARs on the base instance
+---
+
+This guide will give you the step by step on how to generate the metagraph JARs directly on your Base instance.
```
[the remark this answers is not in the document]
Comment (IPadawans) [reply sdk/guides/setup-a-metagraph/base-instance/03-generating-metagraph-jars.md:6]: updated

File: sdk/guides/setup-a-metagraph/base-instance/03-generating-metagraph-jars.md [hunk 13]
```diff
@@ -0,0 +1,103 @@
...
+
+- Return to the root directory:
+
+```scala
+cd ..
+```
+
+## Creating the metagraph project
+
+- Install the coursier and giter8 packages:
```
[the remark this answers is not in the document]
Comment (IPadawans) [reply sdk/guides/setup-a-metagraph/base-instance/03-generating-metagraph-jars.md:54]: Updated

File: sdk/guides/setup-a-metagraph/building-metagraph-instances/01-building-metagraph-l0-instances.md [hunk 14]
```diff
@@ -9,6 +9,10 @@ Let’s create our Metagraph L0 😎, our L0 will need you to set up at least 3
  Remember to replace the parameters that start with `:something...` with your own specific values 
 :::
 
+:::note
+ METAGRAPH_ID it's the metagraphID generated when [Configuring Base Instance](/sdk/guides/setup-a-metagraph/base-instance/configuring-base-instance)
```
[the remark this answers is not in the document]
Comment (IPadawans) [reply sdk/guides/setup-a-metagraph/building-metagraph-instances/01-building-metagraph-l0-instances.md:13]: Updated

File: sdk/guides/setup-a-metagraph/building-metagraph-instances/01-building-metagraph-l0-instances.md [hunk 15]
```diff
@@ -111,52 +113,41 @@ export CL_PASSWORD=":file_password_1"
...
-scp -i ":your_ssh_key_file.pem" :p12_file.p12 your_instance:your_path
+scp -i "MyKeypair.pem" :p12_file.p12 your_instance:code
 ```
 
 - Let's call our `p12` file in this example of `metagraph-file-1.p12`
 
 **GLOBAL L0**
 
-- Let’s create a new directory to run the global L0
+- Let’s copy the `.p12` file to global-l0 directory
```
[the remark this answers is not in the document]
Comment (IPadawans) [reply sdk/guides/setup-a-metagraph/building-metagraph-instances/01-building-metagraph-l0-instances.md:148]: Fixed

File: sdk/guides/setup-a-metagraph/building-metagraph-instances/02-building-currency-l1-instances.md (hunk 1 above)
Comment (IPadawans) [reply sdk/guides/setup-a-metagraph/building-metagraph-instances/02-building-currency-l1-instances.md]: Fixed

File: sdk/guides/setup-a-metagraph/building-metagraph-instances/03-building-data-l1-instances.md (hunk 2 above)
Comment (IPadawans) [reply sdk/guides/setup-a-metagraph/building-metagraph-instances/03-building-data-l1-instances.md]: Fixed

**Pull request merged**